rinja_derive/src/config.rs

```diff
@@ -360,6 +360,26 @@ impl<'a> RawSyntax<'a> {
             }
         }
 
+        for end in [syntax.block_end, syntax.expr_end, syntax.comment_end] {
```

Comment:
I think it should be checked for both starters and enders, not just enders (or starters in the previous version).

Reply:
It's not needed, because we do not try to understand text blocks. The parsing will work with any starting delimiter. I would not like to introduce restrictions needlessly.

Reply:
Fair enough. :+1: